pkg/image/apiserver/admission/imagepolicy/imagereferencemutators/pods.go

```diff
@@ -8,31 +8,15 @@ import (
 	batchv2alpha1 "k8s.io/api/batch/v2alpha1"
 	kapiv1 "k8s.io/api/core/v1"
 	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/kubernetes/pkg/apis/apps"
 	"k8s.io/kubernetes/pkg/apis/batch"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
-
-	appsapiv1 "github.com/openshift/api/apps/v1"
-	securityapiv1 "github.com/openshift/api/security/v1"
-	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
-	securityapi "github.com/openshift/origin/pkg/security/apis/security"
 )
 
-type ContainerMutator interface {
-	GetName() string
-	GetImage() string
-	SetImage(image string)
-}
-
-type PodSpecReferenceMutator interface {
-	GetContainerByIndex(init bool, i int) (ContainerMutator, bool)
-	GetContainerByName(name string) (ContainerMutator, bool)
-	Path() *field.Path
-}
-
 // GetPodSpecReferenceMutator returns a mutator for the provided object, or an error if no
 // such mutator is defined.
 func GetPodSpecReferenceMutator(obj runtime.Object) (PodSpecReferenceMutator, error) {
```

Comment:
I expected this file to land somewhere in `pkg/cmd/openshift-kube-apiserver/admission`, but it looks like the right idea.